src/components/Characters/Fox/Fox.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import FoxArtWork from './FoxArtWork';
+import { CONFUSED } from '../../../constants/characterMoods';
+import ConfusionMarks from './ConfusionMarks';
+import { FEMALE } from '../../../constants/genders';
+import GenderFlower from './GenderFlower';
+import { gender, mood } from '../../../types';
+import Wrapper from '../Wrapper';
+
+function Fox({ gender = FEMALE, mood = CONFUSED }) {
```

Comment:
MALE as a default :)

Reply:
Fixed in 75e07f6e7f3204d1b0c1c24d058b2851629e456a